Framework/Core/src/WorkflowSerializationHelpers.cxx

```diff
@@ -277,34 +298,28 @@ struct WorkflowImporter : public rapidjson::BaseReaderHandler<rapidjson::UTF8<>,
   {
     enter("END_OBJECT");
     if (in(State::IN_INPUT)) {
-      if (!inputHasDescription && !inputHasSubSpec) {
-
-        DataDescriptorMatcher expectedMatcher00{
-          DataDescriptorMatcher::Op::And,
-          OriginValueMatcher{origin.str},
-          std::make_unique<DataDescriptorMatcher>(
-            DataDescriptorMatcher::Op::And,
-            DescriptionValueMatcher{ContextRef{1}},
-            std::make_unique<DataDescriptorMatcher>(
-              DataDescriptorMatcher::Op::And,
-              SubSpecificationTypeValueMatcher{ContextRef{2}},
-              std::make_unique<DataDescriptorMatcher>(DataDescriptorMatcher::Op::Just,
-                                                      StartTimeValueMatcher{ContextRef{0}})))};
-
-        dataProcessors.back().inputs.push_back(InputSpec({binding}, std::move(expectedMatcher00)));
-      } else if (inputHasDescription) {
-        if (inputHasSubSpec) {
-          dataProcessors.back().inputs.push_back(InputSpec(binding, origin, description, subspec, lifetime, inputOptions));
-        } else {
-          dataProcessors.back().inputs.push_back(InputSpec(binding, {origin, description}, lifetime, inputOptions));
+      auto buildMatcher = [](auto& nodes) -> std::unique_ptr<DataDescriptorMatcher> {
+        auto lastMatcher =
+          std::make_unique<DataDescriptorMatcher>(DataDescriptorMatcher::Op::Just,
+                                                  StartTimeValueMatcher(ContextRef{0}));
```

Comment:
`ContextRef{0}` is always revered for start time value matching, right?

Reply:
Yes. We should probably change it to `data_matcher::STARTTIME_POS`.

Reply:
ok, will do